machine/wnc_x86_64/i2ctools/i2ctools-wnc.patch

```diff
@@ -0,0 +1,287 @@
+diff --git a/sys_eeprom/onie_tlvinfo.c b/sys_eeprom/onie_tlvinfo.c
```

Comment:
Okay, so now I know if I want to modify code with onie-syseeprom, I will switch to busybox, and write difference to a busybox patch file.